src/Algebra/Graph/AdjacencyIntMap/Algorithm.hs

```diff
@@ -206,32 +241,96 @@ dfs vs = concatMap flatten . dfsForestFrom vs
 -- 'isSubgraphOf' ('vertices' $ reachable x y) y == True
 -- @
 reachable :: Int -> AdjacencyIntMap -> [Int]
-reachable x = concat . bfs [x]
+reachable x = dfs [x]
```

Comment:
Interesting that switching back to DFS gives us another performance improvement `reachable: 0.83` :)

Reply:
Yeah, I was happy to see that. I felt intuitively that something must be wrong if bfs is performing better for reachable than dfs!